zkevm-circuits/src/evm_circuit.rs

```diff
@@ -475,4 +484,81 @@ mod evm_circuit_stats {
             |_, state, _| state.get_step_height_option().unwrap(),
         );
     }
+
+    /// This function prints to stdout a table with the top X ExecutionState
+    /// cell consumers of each EVM Cell type.
+    ///
+    /// Run with:
+    /// `cargo test -p zkevm-circuits --release get_exec_steps_occupancy
+    /// --features test -- --nocapture --ignored`
+    #[ignore]
+    #[test]
+    fn get_exec_steps_occupancy() {
+        let mut meta = ConstraintSystem::<Fr>::default();
+        let circuit = EvmCircuit::configure(&mut meta);
+
```

Comment:
Can we print some explanation on the meaning of headers? Or maybe point to the document of `StateReportRow`.

` total_avaliable_cells | unused_cells | cells | top_height | used columns (Max: 71) | Utilization`

Reply:
What explanation were you thinking about? I thought the names were explanatory enough. Would appreciate some extra comments on what you mean.

Reply:
The meanings of the headers were not obvious to me. I added the comments in the `StateReportRow` based on my understanding from `Instrument::analyze`.